docs/UserGuide.md

```diff
@@ -311,8 +446,19 @@ _Written by: Yong Chin Han_
 
 ## FAQ
 
-**Q**: How do I transfer my data to another computer?
+**Q**: How do I view commands while the app is running? 
+* **A**: Type `help` command to view all the commands!
+
+**Q**: Can I directly input values into the duke.txt file? 
+* **A**: It is strongly advisable not to edit the duke.txt file as that could disrupt Moolah Manager's retrieval of the data.
```

Comment:
Good disclaimer here. Do u also want to add "If you want to edit entries, it is better to use the Edit command"?